src/components/Issue/CreateIssue.tsx

```diff
@@ -0,0 +1,269 @@
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  TextField,
+  DialogActions,
+  FormControl,
+  FormHelperText,
+  MenuItem,
+  Select,
+} from "@material-ui/core";
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import AddBoxIcon from "@material-ui/icons/AddBox";
+import { AppDispatch, RootState } from "store/store";
+import "./CreateIssue.css";
+import { createIssue } from "common/actions";
+import { IIssue, IUsers } from "common/models";
+
+interface CreateIssueState {
+  title: string;
+  description: string;
+  open: boolean;
+  charactersLeft: number;
+  assignee: string;
+  titleError: string | null;
+  descriptionError: string | null;
+  assigneeError: string | null;
+  isSubmitting: boolean;
+}
+
+interface CreateIssueProps {
+  issues: Array<IIssue>;
+  users: Array<IUsers>
+  dispatch: AppDispatch;
+}
+
+class CreateIssue extends Component<CreateIssueProps, CreateIssueState> {
+  constructor(props: CreateIssueProps) {
+    super(props);
+
+    this.state = {
+      title: "",
+      description: "",
+      open: false,
+      charactersLeft: 255,
+      assignee: "",
+      titleError: "Please enter a title between 3 and 50 characters!",
```

Comment:
Error messages appear in two places in this file. They can be extracted a constant

Reply:
done